tests/fixtures/simple_app_with_overrides/node_modules/file.rb

```diff
@@ -0,0 +1 @@
+Foo
```

Comment:
nit: for this functionality we probably don't need these files. For test cases I was thinking:
- One integration test with one app that has three cases:
  - A pack that has a complete namespace override
  - A pack with a partial namespace override
  - The root having a partial namespace override
  - maybe something else?

We could also have a unit test for these but I've been liking the integration tests because they allow a lot of flexibility in how the internal structure is represented. Unit tests might be good here if there are various edge cases.

Reply:
Oh great call out, thanks for reminding me! I probably just would have kept them otherwise. This is the mess you get when you copy/paste file folders 😂

Reply:
Do you want these overrides to be specified on a per-pack basis then? I think that is a feasible change for this PR, I just want to be clear before doing the implementation needed.

Reply:
@mclark and I chatted over slack (in the Ruby/Rails modularity slack server, [linked in the README](https://github.com/alexevanczuk/packs#kudos)) and we're thinking putting this into `packwerk.yml`/`packs.yml` makes sense for the first iteration at least.

The main reasoning for this is that we felt like it's valuable to consolidate autoload configuration into one place, tying autoloading more to the application as a whole rather than each package. Either way, I think the decision here is pretty reversible – I could see `packs` allowing users to choose one or the other and having `packs` double-read the information.